packages/textarea/src/styled.tsx

```diff
@@ -1,12 +1,44 @@
-import styled from 'styled-components'
+import styled, {DefaultTheme} from 'styled-components'
 import {getTextInputStyle} from '@smashing/text-input'
-import {TextareaProps} from './types'
+import {TextareaProps, TextareaAppearance} from './types'
 import {Text} from '@smashing/typography'
 import {getValueWithUnit} from '@smashing/theme'
 
 type InputProps = TextareaProps &
   Required<Pick<TextareaProps, 'borderRadius' | 'appearance' | 'full'>>
 
+const getTextareaStyle = (
```

Comment:
Sadly, the component is getting a little ugly on Firefox:
<img width="224" alt="Zrzut ekranu 2019-11-19 o 09 12 39" src="https://user-images.githubusercontent.com/8265332/69128653-14a1f600-0aad-11ea-9b3b-e57e9f03961d.png">
You can experiment with Mozilla's attributes (https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) or consider using JavaScript (in a form own or 3rd party code)

Reply:
@hzub I used a library, now it should look fine on every browser

Reply:
ok, cool! @Idered - thoughts?